resolwe/storage/connectors/localconnector.py

```diff
@@ -16,14 +16,17 @@ class LocalFilesystemConnector(BaseStorageConnector):
     def __init__(self, config: dict, name: str):
         """Connector initialization."""
         super().__init__(config, name)
+        self.required_settings = ["path"]
+        self._validate_config()
```

Comment:
It would be great if this is called automatically when the connector is imported.